content/doc/developer/architecture/extensions.adoc

```diff
@@ -2,4 +2,12 @@
 title: Extensions
 layout: developersection
 wip: true
----
\ No newline at end of file
+---
+
+Jenkins defines extension points, which are interfaces or abstract classes that model an aspect of its behavior. The interface or class defines contracts of what needs implementation, and Jenkins allows plugins to contribute those implementations.
```

Comment:
```suggestion
Jenkins defines extension points, which are interfaces or abstract classes that model an aspect of its behavior.
The interface or class defines contracts of what needs implementation, and Jenkins allows plugins to contribute those implementations.
```
I saw that the earlier modification was to start another run (since the previous one had failed), but would it be possible to update this back to be two separate lines?  Thanks!